simulation/simulation_test.go

```diff
@@ -544,12 +562,14 @@ func TestAppSimulationAfterImport(t *testing.T) {
 	)
 	require.NoError(t, err)
 
+	// Initialize the new app with the exported genesis state of the first run
 	t.Log("Importing genesis into the new app")
 	newSimApp.InitChain(abci.RequestInitChain{
 		ChainId:       SimAppChainID,
 		AppStateBytes: exported.AppState,
 	})
 
+	// Run simulation on the new app
```

Comment:
Not related to this line, but shouldn't it be `require.NoError(t, simErr)`
<img width="857" alt="image" src="https://github.com/user-attachments/assets/c8beefef-1651-48bc-8e00-ec9c83aeb716" />